jupyter_server/extension/application.py

```diff
@@ -354,20 +352,23 @@ def _prepare_templates(self):
             })
         self.initialize_templates()
 
-    @staticmethod
-    def initialize_server(argv=[], load_other_extensions=True, **kwargs):
+    @classmethod
+    def initialize_server(cls, argv=[], load_other_extensions=True, **kwargs):
         """Get an instance of the Jupyter Server."""
         # Get a jupyter server instance
         serverapp = ServerApp.instance(**kwargs)
+        # Add extension to jpserver_extensions trait in server.
+        mod = cls._jupyter_server_extension_paths()[0]['module']
```

Comment:
Is the variable meant to be used?

Reply:
It *was* being used as the key to the dictionary in the next line, but I've changed this logic a bit so this no longer applies. Resolving.